source/ui/input-text-model.cpp

```diff
@@ -0,0 +1,43 @@
+#include <iostream>
+
+#include "core/conversions.hpp"
+#include "ui/input-text-model.hpp"
+
+InputTextModel::InputTextModel(QQmlContext* context, MemoryAccess memoryAccess)
+: QObject()
+, context(context)
+, start(0)
+, maxLength(20)
```

Comment:
This produces a compiler error on my machine. I think assigning it inside the constructor would work.